core/invoker/src/main/scala/whisk/core/containerpool/docker/ProcessRunner.scala

```diff
@@ -52,17 +52,75 @@ trait ProcessRunner {
         case _                 => None
       }
 
-      (process.exitValue(), out.mkString("\n"), err.mkString("\n"), scheduled)
+      (ExitStatus(process.exitValue()), out.mkString("\n"), err.mkString("\n"), scheduled)
     }).flatMap {
-      case (0, stdout, _, scheduled) =>
+      case (ExitStatus(0), stdout, _, scheduled) =>
         scheduled.foreach(_.cancel())
         Future.successful(stdout)
-      case (code, stdout, stderr, scheduled) =>
+      case (exitStatus, stdout, stderr, scheduled) =>
         scheduled.foreach(_.cancel())
-        Future.failed(ProcessRunningException(code, stdout, stderr))
+        timeout match {
+          case t: FiniteDuration if exitStatus.terminatedBySIGTERM =>
```

Comment:
`t` is unused so better to use
```suggestion
          case _: FiniteDuration if exitStatus.terminatedBySIGTERM =>
```

Reply:
Good catch. I missed that because I copied from above...